lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -339,6 +339,8 @@ def setUp(self):
 
         self.coeff_names = ["gamma", "delta", "a", "beta"]
 
+    @ManageWarnings(
+        ignored_messages=["Collapsing a non-contiguous coordinate."])
```

Comment:
In this file the same warning message is ignored many times.  Is there a good reason not to declare this as a variable (collapse_msg="Collapsing a non-contiguous coordinate.") rather than constantly repeating the string?

Reply:
I'm not sure what advantage declaring this as a Global variable has apart from meaning you have to scroll to the top of the code to find out what is it but I can change it if you think this is really important.

Reply:
I was thinking in terms of what happens if the string corresponding to this error message ever changes slightly, say in a future version of iris.  And similarly for the other repeated strings.

Reply:
I've added a Global variable to the unit tests although I'm not convinced this change is necessary.